<launch>    
  <arg name="world_frame" default="/world" />
  <arg name="body_frame" default="/base_link" />
  <arg name="use_tf" default="false" />
  <arg name="loop_rate" default="40"/>
  <arg name="mass" default="1.2"/>
  <arg name="cutoff_freq" default="40"/>

    <node name="control_manager" pkg="labrom_mav_control" type="velocity_node" output="screen">
      <!-- Remapping topics-->
      <remap from="cmd_attitude" to="/vrep/cmd_attitude"/>
      <remap from="cmd_thrust" to="/vrep/cmd_thrust"/>    
      <remap from="odometry" to="/vrep/odometry"/>   
      <remap from="cmd_vel" to="/cmd_vel"/>

      <!-- Loading parameters -->
      <rosparam command="load" file="$(find labrom_mav_control)/params/vrep.yaml"/>
      <rosparam subst_value="true">
        world_frame: $(arg world_frame)
        body_frame: robot
        use_tf: $(arg use_tf)
        loop_rate: $(arg loop_rate)
        mass: $(arg mass)
        cutoff_freq: $(arg cutoff_freq)
      </rosparam>

    </node>
</launch>
